### PR TITLE
feat:  add required to bankInstructions. fix id mock formatting

### PR DESCRIPTION
**Pull request by eweise**
@johntowell @bondricek

### CODE REVIEW

**Comment by eweise** [comment]
bump

**Comment by johntowell** [comment]
👍  will this break platform?

File: api/v3/schema/profile.res.json [hunk 1]
```diff
@@ -273,7 +273,8 @@
     "addresses",
     "emails",
     "userStatus",
-    "accounts"
+    "accounts",
+    "bankInstructions"
```
Comment (eweise) [inline api/v3/schema/profile.res.json]: oops actually bankinstructions under wrong required section. fixing.

**Comment by eweise** [comment]
It shouldn't break platform

**Pull request merged**